Audio-Levels/Audio-Levels.xcodeproj/project.pbxproj

```diff
@@ -358,6 +358,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=*]" = i386;
```

Comment:
Why do we needs this now?

Reply:
i386 is no longer supported by our sdk. A couple of samples like Archiving and Broadcast were failing because of that. So I excluded it from all samples, in case someone starts their app from one of these samples and finds out after building issues

Reply:
Thanks!